packages/transactions/README.md

```diff
@@ -1,9 +1,18 @@
 # `@helium/transactions`
 
-> TODO: description
+Construct and serialize transaction primatives from their protobuf definitions.
 
-## Usage
+## Token Burn V1
+```ts
+// create transcation
+ const transaction =  new TokenBurnV1({
+    payer: 'payer_address',
+    payee: 'payee_address',
+    amount: 10,
+    nonce: 1,
+    memo: 1234567890,
+})
 
-```
-// TODO: DEMONSTRATE API
+// sign transaction
+const signedTransaction = await transaction.sign({ payer: 'payer_address' })
```

Comment:
i think the argument to `payer` here would be the full keypair

Reply:
Ah let me fix it